saas/mic-hex-escs/domain/src/main/java/com/sokima/saas/mic/hex/escs/domain/transformer/clean/mapper/Tuple2CleanedCartMapper.java

```diff
@@ -0,0 +1,43 @@
+package com.sokima.saas.mic.hex.escs.domain.transformer.clean.mapper;
+
+import com.sokima.lib.ecommerce.java.domain.model.Cart;
+import com.sokima.lib.ecommerce.java.domain.model.record.CartRecord;
+import com.sokima.saas.mic.hex.escs.domain.payload.clean.CleanCartPayload;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.factory.Mappers;
+
+import reactor.util.function.Tuple2;
+
+@Mapper
+public interface Tuple2CleanedCartMapper {
+
+    Tuple2CleanedCartMapper INSTANCE = Mappers.getMapper(Tuple2CleanedCartMapper.class);
+
+    @Mapping(target = "cartId", source = "t1.cartId")
+    @Mapping(target = "userId", expression = "java(input.getT2().cartId())")
```

Comment:
1. why `userId` maps to`cartId` ? it may be different

2. such mapping could be simplified
```suggestion
    @Mapping(target = "userId", source= "t2.cartId")
```

Reply:
Changed to cartId, but couldn't be simplified, because "Cart has no properties"